backend/models/office_hours/ticket.py

```diff
@@ -39,6 +39,7 @@ class OfficeHoursTicket(OfficeHoursTicketDraft):
     caller_notes: str = ""
     called_at: datetime | None = None
     closed_at: datetime | None = None
+    oh_event: OfficeHoursEvent
```

Comment:
Moved this from Details model so that the `oh_event` field in the to_model() method would avoid having to create a `OfficeHoursEventPartial`. I'm not sure if this is the best way to do it as we typically want the relationship fields in the details model. Please let me know thoughts!